requirements-dev.txt

```diff
@@ -3,3 +3,4 @@ git+https://github.com/spacetelescope/stsci.tools.git#egg=stsci.tools
 astropy>=0.0.dev0
 git+https://github.com/spacetelescope/stwcs.git#egg=stwcs
 git+https://github.com/astropy/astroquery.git#egg=astroquery
+numpy>=0.0.dev0
```

Comment:
This isn't quite right, see https://github.com/spacetelescope/romancal/pull/695#discussion_r1189078751 for details.